internal/services/api/api_service.go

```diff
@@ -82,6 +82,10 @@ func RouterHandlerFunc(base util.AbsolutePath, lister accounts.AccountList, log
 	r.Handle(ToPath("inspect"), PostInspectHandlerFunc(base, log)).
 		Methods(http.MethodPost)
 
+	// POST /api/inspect/remote/$GUID
+	r.Handle(ToPath("inspect", "remote", "{guid}"), PostInspectRemoteHandlerFunc(base, log, lister)).
+		Methods(http.MethodPost)
```

Comment:
The account name could be a URL parameter, like `/api/inspect/remote/$ACCOUNT/$GUID` (I see below that it is in the body)

Reply:
➕ ^, it'll be a more consistent and standard approach to have the request target pointers to be included in the path and not in the body

Reply:
Thanks, I'll change it!